src/common/api.js

```diff
@@ -27,7 +27,11 @@ export function callApi(path) {
 export function* fetchApi(requestTypes, path, action) {
   try {
     path = format(path, action)
-    const data = yield call(callApi, path);
+    let option = {}
```

Comment:
@vinhkq  I don't understand why you insert `action.data.title !== ""`, a such too specific condition, here.